Duplicati/CommandLine/BackendTester/Program.cs

```diff
@@ -225,6 +225,28 @@ static bool Run(List<string> args, Dictionary<string, string> options, bool firs
                 bool skipOverwriteTest = Library.Utility.Utility.ParseBoolOption(options, "skip-overwrite-test");
                 bool trimFilenameSpaces = Library.Utility.Utility.ParseBoolOption(options, "trim-filename-spaces");
 
+                long throttleUpload = 0;
+                if (options.TryGetValue("throttle-upload", out string throttleUploadString))
+                {
+                    if (!(backend is IStreamingBackend) || disableStreaming)
+                    {
+                        Console.WriteLine("Warning: Throttling is only supported in this tool on streaming backends");
+                    }
+
+                    throttleUpload = long.Parse(throttleUploadString);
+                }
+
+                long throttleDownload = 0;
+                if (options.TryGetValue("throttle-upload", out string throttleDownloadString))

```

Comment:
Should this be trying to access the value for `"throttle-download"` instead of `"throttle-upload"`?

Reply:
Yes, it should.